webapp/src/blocks.tsx

```diff
@@ -17,6 +17,7 @@ import * as dialogs from "./dialogs";
 import * as blocklyFieldView from "./blocklyFieldView";
 import { CreateFunctionDialog } from "./createFunction";
 import { initializeSnippetExtensions } from './snippetBuilder';
+import * as cmds from "./cmds"
```

Comment:
Import change here and below can be reverted

Reply:
Removed unused imports [8105a6e](https://github.com/microsoft/pxt/pull/10551/commits/8105a6e3b718b7a64e3077fbe0f2fc73824c8ec0)